leaf-ui/rappid-extensions/AnalysisModels.js

```diff
@@ -92,6 +85,25 @@ var ConfigModel = Backbone.Model.extend({
     },
 });
 
+/**
+ * Analysis Parameters 
+ */
+
+var ConfigAnalysis = Backbone.Model.extend ({ 
+    initialize : function(options){
```

Comment:
I think we might need the action param in here too @amgrubb can you confirm?

Reply:
@leshghi you mean to distinguish between single path and next states? Yes, we should add the action parameter.

Reply:
Good catch.